tests.lisp

```diff
@@ -1,7 +1,4 @@
-;;;; -*- Mode: Lisp; indent-tabs-mode: nil -*-
-
-(defpackage :split-sequence/tests
```

Comment:
Since you remove the `DEFPACKAGE SPLIT-SEQUENCE/TESTS` form altogether, are you sure that this file compiles properly? If you tell Genera to use `Package: SPLIT-SEQUENCE/TESTS` then where is this package defined?